freshmaker/errata.py

```diff
@@ -281,7 +278,11 @@ def get_docker_repo_tags(self, errata_id):
             for repo in repos:
                 tags = repos[repo]['tags']
                 repo_tags[build_nvr][repo] = tags
-        return repo_tags
+
+        tags = dict()
+        for repos in repo_tags.values():
+            tags.update(repos)
+        return tags
```

Comment:
Looks like all this and the old code can be rewritten as:

```python
        return {
            repo: repo_data['tags']
            for build_data in response.values()
            for repo, repo_data in build_data['docker']['target']['repos'].items()
        }
```

Reply:
agreed, this is a much cleaner way to do this. changes made